test/model/test/common/include/model/test/wiredtiger_util.h

```diff
@@ -123,6 +123,13 @@ void wt_txn_set_commit_timestamp(WT_SESSION *session, model::timestamp_t commit_
  */
 model::data_value wt_txn_get(WT_SESSION *session, const char *uri, const model::data_value &key);
 
+/*
+ * wt_txn_get_ext --
```

Comment:
If I understand correctly, the addition here over `wt_txn_get` is that we have a way to express errors. 

Looking at the implementation of `wt_txn_get`, we're just silently discarding WT errors to return an empty value. Maybe that's OK because it's test code, but for me, it raises the question of whether we should replace all uses of `wt_txn_get` with `wt_txn_get_ext`.

Again, this is all test code so this shouldn't hold up the PR.

Reply:
I assumed original `wt_txn_get` semantics were somehow needed behavior and just forked the method, but thanks to your comment I was able to determine that 1) the ROLLBACK ignore case is not even used and 2) in fact my changes replaced the method's only caller, so it's dead code in my branch.

Now I realize there isn't some hypothetical bunch of other callsites I might break, I can do something much simpler. I'll quickly remove the original and rename my version to replace it.